test/test-async-functions.js

```diff
@@ -70,8 +70,11 @@ describe("browser-polyfill", () => {
 
         return window.browser.tabs.query({active: true}).then(
           () => fail("Expected a rejected promise"),
-          (err) => equal(err, fakeChrome.runtime.lastError,
-                         "Got the expected error in the rejected promise")
+          (err) => {
+            instanceOf(err, window.Error, "Expected the error to be an instance of Error");
+            equal(err.message, fakeChrome.runtime.lastError.message,
```

Comment:
```suggestion
            equal(err, new window.Error(fakeChrome.runtime.lastError.message),
```

I had also tried the above but it kept saying that `expected [Error: …] to equal [Error: …]` even if `err.constructor === window.Error`. Maybe it compared other properties as well, not just `.message`